db/migrate/20181029114503_set_rating_url_to_menu.rb

```diff
@@ -0,0 +1,8 @@
+class SetRatingUrlToMenu < ActiveRecord::Migration[5.1]
+  def self.up
+
+    menu_link = MenuLink.create :community_id => 1, :sort_priority => 0
```

Comment:
@noskovgleb are you sure that it is a right decision to use plain id?
I think it's better to use `Community.first.id`, for example.
I also don't see why should it be a migration.

Reply:
You are right, community id should be added like you described
Do you think it would be better to use seeds mechanism?